### PR TITLE
VATRP-416 Added global settings under audio to mute speaker and microphone

### PR DESCRIPTION
**Pull request by zackm0571**

### CODE REVIEW

**Comment by patrickmwatson** [comment]
+1 LGTM when travis ci build succeeds I'll approve.

**Comment by ianblenke** [comment]
You are going to want to cherry-pick a60e43db3d6d518cd38d73921d82cc394a77bad8 and 3b3ef4a2db0c9fd1be7bc5e369a1a9e441067ea0 from master to get this build to go green.

**Comment by shareefalis** [comment]
Uh no, don't do that , use 
git pull https://github.com/VTCSecureLLC/ace-ios.git

File: submodules/linphone [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit b65516c163831c43a5e7a73184a6fbc036e7931e
```
Comment (shareefalis) [inline submodules/linphone]: uh why are we changing submodules.. keep it the same..

**Comment by ianblenke** [comment]
Looks like conflicts now. Sorry Zack, can you revisit this? Thanks!

File: submodules/linphone [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 057bf0430a202586a396d3b625fdc4d5338e1686
```
Comment (shareefalis) [inline submodules/linphone]: -1, 057bf0430a202586a396d3b625fdc4d5338e1686

cd submodules/linphone
git checkout 057bf0430a202586a396d3b625fdc4d5338e1686
cd ../.. 
git add submodule/linphone
git commit
 git push

**Comment by shareefalis** [comment]
+9001!!

**Comment by ianblenke** [comment]
+1 LGTM

**Pull request merged**